src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,21 @@
-export const UserInfo = () => {};
+import React from 'react';
+import './UserInfo.scss';
+
+import { User } from '../../types/User';
+
+type Props = {
```

Comment:
Consider renaming `Props` to `UserInfoProps` for better readability and clarity. 🤓